presto-main/src/main/java/com/facebook/presto/HashPagePartitionFunction.java

```diff
@@ -32,15 +33,21 @@
 import static com.google.common.base.Preconditions.checkNotNull;
 import static com.google.common.base.Preconditions.checkState;
 
-public final class HashPagePartitionFunction
+public class HashPagePartitionFunction
         implements PagePartitionFunction
 {
-    private final int partition;
+    private final int partition; // partition idx
```

Comment:
This comment is unnecessary. The variable name should be self-explanatory. We should change the variable name instead if its purpose is not clear.